lib/httpClient.ts

```diff
@@ -4,13 +4,15 @@ import { Signer, Wallet } from 'ethers'
 import { JsonRpcProvider } from 'ethers/providers'
 import { arrayify, keccak256 } from 'ethers/utils'
 import { Claim, IRole } from './LoginStrategy.types'
+import { Policy } from 'cockatiel'
 
 export class CacheServerClient {
   private signer: Signer
   private httpClient: AxiosInstance
   private provider: JsonRpcProvider
   private failedRequests: Array<(token: string) => void> = []
   private isAlreadyFetchingAccessToken: boolean = false
+  public address: string
```

Comment:
I would set to be `readonly` (the `signer`, `provider` and `httpClient` should probably be as well)

Reply:
👍